harness/tests/test_custom_searcher.py

```diff
@@ -0,0 +1,217 @@
+import logging
+import subprocess
+import tempfile
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Sequence
+
+import pytest
+from numpy import float64
+from urllib3.connectionpool import HTTPConnectionPool, MaxRetryError
+
+from determined.common.api import bindings
+from determined.e2e_tests.tests.fixtures.custom_searcher.searchers import (
```

Comment:
tests are failing because of this. I'm not sure how to get the fixtures from e2e_tests. Will take a look on Monday